docs/symptom-survey/contingency-tables.md

```diff
@@ -158,26 +213,27 @@ contact us at <delphi-survey-info@lists.andrew.cmu.edu>.
+| `pct_barrier_not_recommended` | Estimated percentage of respondents who say they are hesitant to get vaccinated because their doctor did not recommend it, among respondents who answered "Yes, probably", "No, probably not", or "No, definitely not" when asked if they would get vaccinated if offered. <br/> **Earliest date available:** 2021-02-08 | V5abc |
+| `pct_barrier_wait_safety` | Estimated percentage of respondents who say they are hesitant to get vaccinated because they want to wait to see if the COVID-19 vaccines are safe, among respondents who answered "Yes, probably", "No, probably not", or "No, definitely not" when asked if they would get vaccinated if offered. <br/> **Earliest date available:** 2021-02-08 | V5abc |
+| `pct_barrier_low_priority` | Estimated percentage of respondents who say they are hesitant to get vaccinated because they think other people need it more than they do, among respondents who answered "Yes, probably", "No, probably not", or "No, definitely not" when asked if they would get vaccinated if offered. <br/> **Earliest date available:** 2021-02-08 | V5abc |
+| `pct_barrier_cost` | Estimated percentage of respondents who say they are hesitant to get vaccinated because they are worried about the cost, among respondents who answered "Yes, probably", "No, probably not", or "No, definitely not" when asked if they would get vaccinated if offered. This series of items was shown to respondents starting in Wave 8. <br/> **Earliest date available:** 2021-02-08 | V5abc |
+| `pct_barrier_distrust_vaccines` | Estimated percentage of respondents who say they are hesitant to get vaccinated because they don't trust COVID-19 vaccines, among respondents who answered "Yes, probably", "No, probably not", or "No, definitely not" when asked if they would get vaccinated if offered. <br/> **Earliest date available:** 2021-02-08 | V5abc |
+| `pct_barrier_distrust_govt` | Estimated percentage of respondents who say they are hesitant to get vaccinated because they don't trust the government, among respondents who answered "Yes, probably", "No, probably not", or "No, definitely not" when asked if they would get vaccinated if offered. <br/> **Earliest date available:** 2021-02-08 | V5abc |
+| `pct_barrier_health_condition` | Estimated percentage of respondents who say they are hesitant to get vaccinated because they have a health condition that may impact the safety of a COVID-19 vaccine, among respondents who answered "Yes, probably", "No, probably not", or "No, definitely not" when asked if they would get vaccinated if offered. <br/> **Earliest date available:** 2021-02-08 | V5abc |
+| `pct_barrier_pregnant` | Estimated percentage of respondents who say they are hesitant to get vaccinated because they are pregnant or breastfeeding, among respondents who answered "Yes, probably", "No, probably not", or "No, definitely not" when asked if they would get vaccinated if offered. <br/> **Earliest date available:** 2021-02-08 | V5abc |
```

Comment:
```suggestion
| `pct_barrier_pregnant` | Estimated percentage of respondents who say they are hesitant to get vaccinated because they are currently, or are planning to be, pregnant or breastfeeding, among respondents who answered "Yes, probably", "No, probably not", or "No, definitely not" when asked if they would get vaccinated if offered. <br/> **Earliest date available:** 2021-02-08 | V5abc |
```